<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fruit_Server</class>
 <widget class="QMainWindow" name="Fruit_Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fruit_Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
background-image: url(:/photo/833691.jpg);
}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1511</width>
      <height>502</height>
     </rect>
    </property>
    <property name="title">
     <string>水果状态栏</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_banana">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>184</y>
       <width>518</width>
       <height>149</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_grape">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>340</y>
       <width>518</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_watermelon">
     <property name="geometry">
      <rect>
       <x>961</x>
       <y>184</y>
       <width>531</width>
       <height>149</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_cucumber">
     <property name="geometry">
      <rect>
       <x>961</x>
       <y>340</y>
       <width>531</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_pear">
     <property name="geometry">
      <rect>
       <x>962</x>
       <y>27</y>
       <width>531</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>27</y>
       <width>207</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/apple.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>184</y>
       <width>207</width>
       <height>149</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/banana.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>50</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/pear.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>751</x>
       <y>184</y>
       <width>207</width>
       <height>149</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/watermelon.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/grape.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>350</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/photo/cucumber.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_apple">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>27</y>
       <width>518</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255,100);</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1041</x>
      <y>34</y>
      <width>425</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>无人超市水果采集系统（客户端）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>12</y>
      <width>250</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>12</y>
      <width>250</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>620</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>620</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>621</y>
      <width>258</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>主机端口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>620</y>
      <width>202</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>主机地址</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP通讯实训项目</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
